src/app/containers/ConsentBanner/index.jsx

```diff
@@ -2,9 +2,41 @@ import React, { useContext } from 'react';
 import { RequestContext } from '../../contexts/RequestContext';
 import Canonical from './index.canonical';
 import Amp from './index.amp';
+import { EventContext } from '../../contexts/EventContext';
+import { buildATIClickParams } from '../ATIAnalytics/params';
+import { sendEventBeacon } from '../ATIAnalytics/beacon';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+export const ConsentBanner = props => {
```

Comment:
Not sure why this is exported twice from this file?

Reply:
Fixed!